src/core/midi/MidiController.cpp

```diff
@@ -80,7 +80,7 @@ void MidiController::processInEvent( const MidiEvent& event, const TimePos& time
 		case MidiControlChange:
 			controllerNum = event.controllerNumber();
 
-			if( m_midiPort.inputController() == controllerNum + 1 &&
+			if( m_midiPort.inputController() == controllerNum &&
 					( m_midiPort.inputChannel() == event.channel() + 1 ||
 					  m_midiPort.inputChannel() == 0 ) )
```

Comment:
```suggestion
			if(m_midiPort.inputController() == controllerNum &&
					( m_midiPort.inputChannel() == event.channel() + 1 ||
					  m_midiPort.inputChannel() == 0 ))
```

Approving regardless of this because the conditional spans multiple lines, but if you want to just accept the change just before merging it's one less if-statement with uncompliant code-style for us to fix later :sweat_smile:

Reply:
done - I have removed stray whitespaces for the entire function